graphics/ipe/Portfile

```diff
@@ -18,40 +19,27 @@ long_description        \
 license                 GPL-3+
 homepage                http://ipe.otfried.org
 platforms               macosx
-master_sites            https://dl.bintray.com/otfried/generic/ipe/${branch}
-distname                ${name}-${version}-src
+
 worksrcdir              ${name}-${version}/src
 
-checksums               rmd160  eaa2b9fc1a923a78b14e1c1c27f8ffc139febe0e \
-                        sha256  a22dcae9cb660f466678a6b568e9fed1b12a3830e49465594d65fc789b0ba725 \
-                        size    1950071
+checksums           rmd160  3aab1c281c938f10c80d29d23c46df963c645b8c \
```

Comment:
Can we match the alignment that was there previously?

Reply:
done